docs/install.rst

```diff
@@ -43,10 +45,31 @@ If you want to install PyScaffold with all official extensions, run::
 
     pip install --upgrade pyscaffold[all]
 
-PyScaffold is also available at `conda-forge`_ and thus can be installed with `conda`_::
+
+Alternative Methods
+===================
+
+It is very easy to get PyScaffold installed with `pip`_, but some people do
+prefer other package managers such as `conda`_ while doing their work.
+
+If you do lots of number crunching or data science in general [#inst2]_ and you already
+rely on `conda-forge`_ packages, you might be attempt the following method::
 
     conda install -c conda-forge pyscaffold
 
+It is also very common for developers to have more then one Python version
+installed in their machines, and a plethora of virtual environments spread all
+over the place… Instead of constantly re-installing PyScaffold in each one of
+these Pythons end venvs, you can use `pipx`_ to do a "minimally-invasive"
```

Comment:
I would rather say virtual environments or introduce it as an abbreviation before.